apps/dashboard/app/views/projects/_template_select.html.erb

```diff
@@ -0,0 +1,11 @@
+  <div class='card-group'>
+    <div class='card'>
+      <div class='card-body'>
+        <%- unless templates.to_a.empty? -%>
+          <div class="field h4">
+            <%= form.select(:template, templates) %>
```

Comment:
Let's make it `required` if we're here making a project from a template. Just to reinforce the use case.

Reply:
will do. Thanks!

Reply:
Also - you can specify `label_class` here in this method. You don't need the wrapper `div`. Not super important, just FYI.